nss-cohort-36/bangazon-client-imagination-station-react
# Lr readme

# Description
Readme.md should be complete

## Type of change
Please delete options that are not relevant.

# Testing Instructions
View Readme and make sure spelling and instructions for downloading are correct.

# Checklist:
